### DISCUSSION TITLE
Community Outreach + Process Ideas

### DISCUSSION THREAD

**Discussion by everettraven** [operator-framework/operator-controller#308] [General]
I've been thinking about how we might be able to increase the visibility of the OLMv1 sub-projects and the work we are doing and wanted to start this discussion to get thoughts and try to come to a consensus on an approach to doing outreach and attempting to grow our community (which might help us get more early adopters!). I'll break this down into 2 sections, "Outreach" and 'Process".

## Outreach
The goal of this section is to make the work we are doing more visible and easily discoverable. This could include (but is not limited to):
- Increasing involvement in larger Slack channels where interest in Kubernetes Operators is prevalent (for example, the `#kubernetes-operators` channel in the Kubernetes Slack). For reference, whenever a new Operator-SDK release is created, it is announced to this channel along with a couple others.
- Reaching out to maintainers of operators to see if they would be interested in adding OLMv1 as a supported installation method
- Participating in mentorship programs. Through our CNCF membership we have the ability to participate in the Google Summer of Code (and other related programs) and the Linux Foundation mentorship program
- Improving new contributor experience of finding good issues to work on (ensuring there are issues labeled "good-first-issue" and the area in which they are targeted - think docs, QA, etc.)
- Participation in OSS events like Hacktoberfest
- Blog posts

## Process
Outreach and growing a healthy community is a lot of work! As part of this discussion I'd like to come to an agreement on how outreach and community growth activities are done so please do share your thoughts and ideas on how we can make this an enjoyable process for everyone involved.

I'll start with my idea - community ambassadors. This could be a volunteer/elected role with a focus on ensuring we are actively participating and taking advantage of outreach and visibility opportunities. They could help ensure there is a good contributor experience, participation in mentoring opportunities, etc.

---

I look forward to continuing this discussion with everyone and coming to an agreement on the best way to grow our community!

**Comment by fgiloux** [comment]
Are making the projects more accessible part of this discussion? I believe that this is key for growing the community but I don't know whether it is in scope of what you have in mind for this discussion.
In this respect I think it is key that a new comer, who knows nothing about OLM to get in 5-10 minutes:
- an idea of what it is good for (value proposition)
- the top level concepts
- a demo of the main use case(s)
- links for deeper dives in some aspects

It is great that there is now some documentation: https://operator-framework.github.io/operator-controller/ 
The entry point of it should however address in my opinion the OLM introduction in 5 to 10 minutes

**Reply by everettraven** [reply]
This is absolutely a part of this discussion. My goal with this discussion is to come up with any and all ideas to improve community growth and come to consensus on a plan of action to make it happen. 

I think you raise some great points and I agree that essentially having a TLDR for OLMv1 is a great way to make the project more accessible.

**Comment by ncdc** [comment]
Thinking about the goals here, this is easily something that can fully occupy 1 or more people's time, if we let it. Given that we probably don't have anywhere near a full person's schedule to use, we have to be more strategic. My proposal: define a story that we want to tell as a series of blog posts, then write and publish them. Everyone involved in OF/OLM can post to their socials if they're willing. That's probably a decent starting place.
